Session-3/CurrencyExchangeApp/CurrencyConverter/ConvertCurrency.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using RestSharp;
+using Newtonsoft.Json;
+
+
+namespace CurrencyConverter
+{
+    public class ConvertCurrency
+    {
+        public dynamic Convert(string from, string to, double value)
+        {
+            string currency = String.Format("{0}{1}", to, from);
+            var client = new RestClient(String.Format("https://api.apilayer.com/currency_data/live?source={0}&currencies={1}", to, from));
+            var request = new RestRequest();
+            request.AddHeader("apikey", "QHbQjcYfLJzxGT3UAFEu2KcQW4GAQWcg");
+            CurrencyObject ApiObject=new CurrencyObject();
+            RestResponse response = client.Get(request);
+            ApiObject = JsonConvert.DeserializeObject<CurrencyObject>(response.Content);
+            if (ApiObject.success==true)
+            {
+                if(to == from)
+                {
+                    Exception SameCurrencyException = new Exception("Enter different currencies to convert");
```

Comment:
why exception if they pass same currency? you can simply return the amount without any convertion